rock_gazebo.orogen

```diff
@@ -49,10 +49,17 @@ task_context "ModelTask" do
     # The covariance matrix to be used as velocity covariance. Default is unset (NaN)
     property 'cov_velocity', '/base/Matrix3d'
 
+    # The acceleration of the model w.r.t. the world frame
+    output_port 'ang_accel_samples', '/base/samples/RigidBodyAcceleration'
```

Comment:
While I can understand the will to export as much data out of gazebo as possible ... it's also a bit strange to only have the angular accelerations. Moreover, angular accelerations is not something that can be measured in the real world.

Maybe having a type specific to the `rock_gazebo` package with the whole acceleration state ? I really don't know why the designers of `RigidBodyAcceleration` only added the linear acceleration ...